src/components/searchform.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../css/searchform.css';
+
+class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: this.props.initialSearchQuery || '',
+    };
+  }
+
+  handleInputChange = (e) => {
+    this.setState({ searchQuery: e.target.value });
+  };
+
+  handleSearch = () => {
+    this.props.onSearch(this.state.searchQuery);
+  };
+
+  handleKeyPress = (e) => {
+    if (e.key === 'Enter') {
+      this.handleSearch();
+    }
+  };
+
+  render() {
+    return (
+      <div className="form-container">
+        <p className="header-element">FIND YOUR MOVIE</p>
```

Comment:
Find Your Movie - please move all the literals to a separate file, like literals.js
for example:
`export const FIND_YOUR_MOVIE = 'Find Your Movie'`

Reply:
Created literlas.js and exported the constant.